### ISSUE TITLE
Datasource provisioning with direct DB connection enabled failed

### ISSUE THREAD

**Issue by sdestivelle** [alexanderzobnin/grafana-zabbix#688]
**Describe the bug**
When setting the dbConnectionEnable to True according to the [documentation](https://alexanderzobnin.github.io/grafana-zabbix/configuration/provisioning/)
The Direct DB Connection is not Enable.

**Expected behavior**
The Direct DB Connection set to True is the web interface.

**Screenshots**
<img width="262" alt="image" src="https://user-images.githubusercontent.com/6033762/53560502-c890d900-3b4c-11e9-9455-a7a6122f784a.png">

**Software versions**

| Grafana | Zabbix | Grafana-Zabbix Plugin |
| ------- | ------ | --------------------- |
| 6.0.0   | 4.0.3  | 3.10.0                 |

**Comment by alexanderzobnin** [comment]
@sdestivelle Yes, got it. This problem caused by wrong config schema migration. In order to solve it, add `schema: 3` to your config:
```yaml
jsonData:
    schema: 3
    username: zabbix
    password: zabbix
    dbConnectionEnable: true
    dbConnectionDatasourceName: MySQL Zabbix
```

**Comment by alexanderzobnin** [comment]
@sdestivelle I fixed it in `3.10.1`, so try to check it out (should work without `schema` field).

**Comment by sdestivelle** [comment]
@alexanderzobnin I will try.
Thanks.
